.github/workflows/node.js.yml

```
@@ -0,0 +1,27 @@
+name: Update data
+
+on:
+  push:
+    branches: [ main ]
```

Let's see if I can get this to trigger in this PR and then I'll remove it before pushing to `main`.

```suggestion
    branches: [ main ]
    pull_request:
      branches:
        - main
```